common-content/en/module/tools/programming-language-concepts/index.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "Programming language concepts"
+headless = true
+time = 120
+facilitation = false
+emoji= "📖"
+objectives=[
+    "Describe what a variable is.",
+    "Describe how a variable relates to a memory location.",
+    "Give examples of common operators.",
+    "Describe the meaning of the `&&` (`and`), `||` (`or`), and `!` (`not`) operators.",
+    "Explain what happens when you call a function.",
+]
++++
+
+Read the learning objectives listed on this page: Bear in mind what you’re trying to achieve while reading this text. If a topic isn’t making much sense, and isn’t in the objectives, you can probably skip over it. If a topic is listed in the objectives, you should keep studying it until you are confident you’ve met the objective.
+
+{{<note type="Reading">}}
+Read chapter 9 of How Computers Work.
+
+Do every exercise listed in the chapter.
+
+You can skip the projects (though you're welcome to try any of them if you have time!).
+{{</note>}}
+
+Check you have achieved each learning objective listed on this page.
+
+{{<note type="Exercise">}}
+Write down an explanation of what a variable is. Make sure to use the term "memory location" in your explanation.
+{{</note>}}
+
+{{<note type="Exercise">}}
+Write down what operators you have used in a programming language.
+
+Next to each, write down whether it's a unary, binary, or ternary operator.
```

Comment:
Grouping and sorting is a good learning tool, so let's tweak this slightly to use it. 
```suggestion
Now sort your operators into three categories: unary, binary, or ternary. Label each operator as a unary, binary, or ternary operator.
```

Reply:
Will make this a Labeller